addons/packages/kapp-controller/0.30.0/bundle/config/schema.yaml

```diff
@@ -0,0 +1,45 @@
+#! schema.yaml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for kapp-controller"
+---
+#@schema/desc "The namespace value used by older templates, will overwrite with top level namespace of present, for backward compatibility"
+namespace: kapp-controller
+#@schema/desc "Configuration for kapp-controller"
+kappController:
+  #@schema/desc "The namespace value used by older templates, will overwrite with top level namespace of present, for backward compatibility"
+  #@schema/nullable
+  namespace: kapp-controller
```

Comment:
Do both this "namespace" field and the one above in line 7, have same descriptions?

Reply:
no, it shud be different. Fixed